src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -465,6 +473,23 @@ public function testDecodeEmptyXml()
         $this->encoder->decode(' ', 'xml');
     }
 
+    protected function getXmlPISource()
```

Comment:
Oh ok great :+1: I hope they won't make you change all other examples for the sake of consistency :P